weave

```diff
@@ -265,8 +271,9 @@ PLUGIN_NAME="$DOCKERHUB_USER/net-plugin"
 OLD_PLUGIN_CONTAINER_NAME=weaveplugin
 CNI_PLUGIN_NAME="weave-plugin-$IMAGE_VERSION"
 CNI_PLUGIN_DIR=${WEAVE_CNI_PLUGIN_DIR:-$HOST_ROOT/opt/cni/bin}
+VOLUMES_LABEL=weavevolumes
 # Note VOLUMES_CONTAINER which is for weavewait should change when you upgrade Weave
-VOLUMES_CONTAINER_NAME=weavevolumes-$IMAGE_VERSION
+VOLUMES_CONTAINER_NAME=$VOLUMES_LABEL-$IMAGE_VERSION
```

Comment:
`VOLUMES_LABEL` and this container name are not related except by coincidence, so this is potentially confusing.

Reply:
Removed the reuse of the variable